pages/glossary/distinguishing-styles.md

```diff
@@ -0,0 +1,11 @@
+--
+title: Distinguishing Styles
+key: distinguishing-styles
+--
+
+Two elements have distinguishing styles if any of the following is true:
+
+- they have [distinguishing borders](#distinguishing-borders)
+- they have [distinguishing box-shadows](#distinguishing-box-shadows)
+- they have different `background-image`,
```

Comment:
What's meant by "different"? Are two images with the same URI different, whether it be a data URI or a URL? Do they have to contain different byte sequences? Do they have to subjectively differ in what they depict?

Reply:
We've added an assumption to the definition.